src/oidcop/oidc/token.py

```diff
@@ -242,12 +239,12 @@ def process_request(self, req: Union[Message, dict], **kwargs):
             _resp["expires_in"] = access_token.expires_at - utc_time_sans_frac()
 
         _mints = token.usage_rules.get("supports_minting")
-        issue_refresh = False
-        if "issue_refresh" in kwargs:
-            issue_refresh = kwargs["issue_refresh"]
-        else:
-            if "offline_access" in scope:
-                issue_refresh = True
+
+        issue_refresh = kwargs.get("refresh_token", False)
+        # The existence of offline_access scope overwrites issue_refresh
+        if "offline_access" in scope:
+            issue_refresh = True
```

Comment:
The logic is reversed here. `issue_refresh` used to override `offline_access`, which IMHO is the correct behavior. Maybe this is better:
```python
        issue_refresh = kwargs.get("refresh_token", None)
        # The existence of offline_access scope overwrites issue_refresh
        if issue_refresh is None and "offline_access" in scope:
            issue_refresh = True
```